docs/getting_started/requirements.md

```diff
@@ -10,26 +10,26 @@ The following server requirements cover both running the software on-premise and
 
     For running on [Ibexa Cloud](https://www.ibexa.co/products/ibexa-cloud), where recommended configuration and support is provided out of the box, see separate [Ibexa Cloud section](#ibexa-cloud-requirements-and-setup) for further reading on its requirements.
 
-The minimal setup requires PHP,  MySQL/MariaDB, Apache/Nginx, Node.js and `yarn`.
+The minimal setup requires PHP, MySQL/MariaDB, Apache/Nginx, Node.js and `yarn`.
 Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached, NFS/EFS/S3 and Solr/Elasticsearch in a [clustered setup](clustering.md).
 
 Using the latest listed version of each product or component is always recommended.
 
 ## Operating system
 
 - Debian 10.x "buster" or Debian 11.x "bullseye"
-- Ubuntu 20.04 "Focal Fossa"
+- Ubuntu 20.04 "Focal Fossa" or higher
 - RHEL / CentOS 8.1+
 
 ## Web server
 
-- Nginx 1.18 or higher
+- Nginx 1.18+
```

Comment:
Should we clarify in cases like this that we mean "any 1.x version higher or equal to 1.18, but not 2.x"? The same goes for "MariaDB 10.3+" and others. I mean, even if Nginx 2 and MariaDB 11 don't exist yet, suddenly one day they are released, and some customers will demand they work from day one, given text like this. Or do you guys feel it's clear?

Reply:
I feel it's clear, but I think it is worth mentioning somewhere.

Reply:
https://github.com/ezsystems/developer-documentation/pull/1826/files#diff-113e92f1aa1bfe8d987f618755d55f4e62bb5158db53286243b9b58746533b85R16

Reply:
Resolved, but see separate punctuation suggestion :)